scala/zio-http/build.sbt

```diff
@@ -0,0 +1,8 @@
+name := "server"
+scalaVersion := "2.13.5"
+
+lazy val `zio-http` = (project in file(".")).settings(
+  libraryDependencies += "io.d11" %% "zhttp" % "1.0.0.0-RC13"
```

Comment:
I'll prefer to wait until the project is out of `release candidate` @whiplash @tusharmath

Reply:
It can make sense. @tusharmath do you have any release plan? even just an idea.

Reply:
The team is trying its best to have something by Q2 2021, but I can't really assure you that v1 will be out by then.

Reply:
no rush ... we will track new release

the purpose of this project is to see how awesome could be tech world ❤️ , better to wait for quality software